UI/src/elements/lsmb-country.js

```diff
@@ -0,0 +1,72 @@
+/** @format */
+
+import { LsmbBaseInput } from "@/elements/lsmb-base-input";
+import { useCountriesStore } from "@/store/countries";
+
+const Select = require("dijit/form/Select");
+const registry = require("dijit/registry");
+
+export class LsmbCountry extends LsmbBaseInput {
+    store = null;
+
+    widgetWrapper = null;
+
+    _boolAttrs() {
+        return ["default_blank"];
+    }
+
+    _widgetRoot() {
+        if (this.widgetWrapper) {
+            return this.widgetWrapper;
+        }
+        this.widgetWrapper = document.createElement("span");
+        this.appendChild(this.widgetWrapper);
+
+        return this.widgetWrapper;
+    }
+
+    async connectedCallback() {
+        let props = this._collectProps();
+
+        if (!this.store) {
+            /* Instantiate *and* initialize store before assigning to
+             * the object property to avoid a race condition.
+             */
+            const store = useCountriesStore();
+            await store.initialize();
+            this.store = store;
+        }
+
+        const options = this.store.items.map((element) => {
+            return { label: element.name, value: element.code };
+        });
+
+        if ("default_blank" in props) {
+            options.unshift({ label: "", value: "" });
```

Comment:
```suggestion
            options.unshift({ code: "", name: "" });
```

which then changes this line to unshift the named properties instead of "label" and "value".